Source/swiftlint/Commands/AutoCorrectCommand.swift

```diff
@@ -17,6 +17,17 @@ struct AutoCorrectCommand: CommandProtocol {
     func run(_ options: AutoCorrectOptions) -> Result<(), CommandantError<()>> {
         let configuration = Configuration(options: options)
         let cache = options.ignoreCache ? nil : LinterCache(configuration: configuration)
+        let indentWidth: Int
+        let useTabs: Bool
+
+        switch configuration.indentationMode {
+        case .tabs:
+            indentWidth = 4
```

Comment:
I'm not sure it makes sense to have an indentWidth when using tabs, but it's required by the formatter - using the previous hardcoded value seems like a safe bet.